test/spec/test_helper/load.bash

```diff
@@ -117,8 +117,17 @@ fi
 
 # Use in setup(), separated out so setup() may be overridden.
 source_stdlib() {
+  local statefile
+  statefile="$(bats_mktemp)"
+  stdlib="$(bats_mktemp)"
+
+  ( cd "${BATS_TEST_DIRNAME%/}/../../" || return 1;
+    terraform init -input=false;
+    terraform apply -input=false -state-out="${statefile}";
+    terraform output -state="${statefile}" content > "${stdlib}")
```

Comment:
This slows test runs down, but it's still under 1 minute real time (~30secs with `make docker_bats`, ~20secs with `make docker_bats_parallel`)